radix-engine/src/model/nodes/resource_manager.rs

```diff
@@ -399,17 +392,14 @@ impl ResourceManager {
                         ),
                     };
                     let bucket_id = system_api
-                        .node_create(HeapRENode::Bucket(Bucket::new(container)))
-                        .map_err(InvokeError::Downstream)?
+                        .node_create(HeapRENode::Bucket(Bucket::new(container)))?
                         .into();
```

Comment:
Can remove the `into()` after question mark. There are a few other occurrences too.

Reply:
`.into()` is required to change `RENodeId` to `BucketId`